src/Symfony/Bundle/FrameworkBundle/Command/TranslationDebugCommand.php

```diff
@@ -223,9 +223,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
                     }
                 }
 
-                if (!\in_array(self::MESSAGE_UNUSED, $states) && $input->getOption('only-unused')
-                    || !\in_array(self::MESSAGE_MISSING, $states) && $input->getOption('only-missing')
-                ) {
+                if (!\in_array(self::MESSAGE_UNUSED, $states, true) && ($input->getOption('only-unused') || $input->getOption('only-missing'))) {
```

Comment:
You changed the condition, was it on-purpose?

Adding the strict param would be only:
```suggestion
                if (!\in_array(self::MESSAGE_UNUSED, $states, true) && $input->getOption('only-unused')
                    || !\in_array(self::MESSAGE_MISSING, $states, true) && $input->getOption('only-missing')
                ) {
```

Reply:
Ah yes indeed, I didn't see that two different consts were used and thought it could be simplified :facepalm: Thanks!